ui/main.qml

```diff
@@ -477,6 +459,10 @@ StatusWindow {
                 }
                 stack.clear()
                 stack.push(splashScreenV2, { runningProgressAnimation: true })
+
+                onboardingStore.appLoaded.connect((keyUid) => {
+                    keychain.saveCredential(keyUid, data.password || data.pin)
```

Comment:
Hmm, will the `data` still be around by the time `appLoaded` has been called?

Reply:
And as Igor noted, the whole module gets destroyed shortly after

Reply:
It's direct connection to the signal, so it's executed when the `appLoaded` is emitted. The module must exist then, I see no risk there.